### PR TITLE
[config reload]Config Reload Enhancement

### PR DESCRIPTION
**Pull request by dgsudharsan**
#### Why I did it
Implementing code changes for https://github.com/sonic-net/SONiC/pull/1203

#### How I did it
Removed the timers and delayed target since the delayed services would start based on event driven approach.
Cleared port table during config reload and cold reboot scenario.
Modified yang model, init_cfg.json to change has_timer to delayed

#### How to verify it
Running regression

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Comment by dgsudharsan** [comment]
Failures are due to dependencies on https://github.com/sonic-net/sonic-utilities/pull/2693 https://github.com/sonic-net/sonic-host-services/pull/45 and https://github.com/sonic-net/sonic-mgmt/pull/7558

File: files/scripts/swss.sh [hunk 1]
```diff
@@ -274,6 +274,8 @@ stop() {
     if [[ x"$WARM_BOOT" != x"true" ]] && [[ x"$FAST_BOOT" != x"true" ]]; then
         /usr/bin/${SERVICE}.sh stop $DEV
         debug "Stopped ${SERVICE}$DEV service..."
+        $SONIC_DB_CLI APPL_DB DEL PORT_TABLE:PortInitDone
```
Comment (qiluo-msft) [inline files/scripts/swss.sh:277]: >DEL [](http://example.com/codeflow?start=30&length=3)

Why DEL is needed? Is it needed before this PR?

Comment (qiluo-msft) [reply files/scripts/swss.sh:277]: This is during `stop` function. Will it be better to DEL in `start` function?

**Review by qiluo-msft — APPROVED** [review]

**Comment by dgsudharsan** [comment]
Depends on https://github.com/sonic-net/sonic-utilities/pull/2693 https://github.com/sonic-net/sonic-host-services/pull/45

**Comment by qiluo-msft** [comment]
Please check conflictions.

**Review by qiluo-msft — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**